### ISSUE TITLE
TFT plotting a single "Predicted" curve in a Multi Target problem

### ISSUE THREAD

**Issue by FrancescoFondaco** [jdb78/pytorch-forecasting#1056]
PyTorch-Forecasting version: 0.10.2
PyTorch version:
Python version: 3.8.5
Operating System: Windows

Hi all,
I'm trying to implement a Multi-Target regression problem using pytorch-forecasting TFT.
As in the image below, despite the code is running smoothly, the plot_prediction() method is showing a single "_actual_" curve and two predicted ones.

![image](https://user-images.githubusercontent.com/107486788/178001989-3ecf2ef6-3240-4ac7-92b0-cce2e16e5080.png)

My dataloader configuration:
`training = TimeSeriesDataSet(
    myData,
    time_idx="Time_idx",
    target=["Target1","Target2"],
    group_ids=["Fund"],
    min_encoder_length=20,  
    max_encoder_length=80,
    min_prediction_length=1,
    max_prediction_length=7,
    static_categoricals=["Fund"],
    static_reals=["Vintage", ....],
    time_varying_unknown_categoricals=[],
    time_varying_unknown_reals=['NAV', .... ],
    target_normalizer=MultiNormalizer(
        normalizers=[GroupNormalizer(),GroupNormalizer()]
    ),
    add_relative_time_idx=False,
    add_target_scales=False,
    add_encoder_length=True,
    #randomize_length=(10,1)
)`

`tft = TemporalFusionTransformer.from_dataset(
    training,
    learning_rate=0.001,
    hidden_size=16,
    attention_head_size=2,
    hidden_continuous_size=8,
    output_size=[7,7],  # 7 quantiles by default
    loss=MultiLoss([QuantileLoss(), QuantileLoss()]),
    log_interval=10, 
    #reduce_on_plateau_patience=4, 
    #optimizer = "sgd" #default 'ranger'
)`

Has anyone encountered this problem?
Thanks,
Francesco
